frontend/utils/format.ts

```diff
@@ -41,23 +41,44 @@ export function commmifyLeft(value: string): string {
   return formatted
 }
 
-export function formatFiat(
+export function formatFiatCurrency(
   value: number,
-  currency: string,
-  locales: string,
-  minimumFractionDigits?: number,
-  maximumFractionDigits?: number,
+  options: {
+    currency?: string,
+    locales?: string,
```

Comment:
np: type to broad, we either handle the locales manually or we might create
something like that: (i18n/i18n.config.ts)
![image](https://github.com/user-attachments/assets/1043d53d-6b85-45a2-b858-59b8a5017493)

```suggestion
    locales?: 'en-US',
```